src/traceability/azure-traceability-helper.ts

```diff
@@ -97,7 +98,7 @@ function getDeploymentResourceUri(aksResourceContext: AksResourceContext): strin
   return `${aksResourceContext.managementUrl}subscriptions/${aksResourceContext.subscriptionId}/resourceGroups/${aksResourceContext.resourceGroup}/providers/Microsoft.Devops/deploymentdetails/${deploymentName}?api-version=2020-12-01-preview`;
 }
 
-async function createDeploymentReport(kubectl: Kubectl, deploymentConfig: DeploymentConfig): Promise<DeploymentReport> {
+async function createDeploymentReport(kubectl: Kubectl, deploymentConfig: DeploymentConfig, status: string): Promise<DeploymentReport> {
```

Comment:
Should we make 'status' - > 'runStatus' or 'deployStatus' ?

Reply:
Changed to `runStatus`.